synapse/models/orgs.py

```diff
@@ -92,18 +92,42 @@ def getBaseModels():
                 ('ou:sic', {'subof': 'int', 'doc': 'Standard Industrial Classification Code'}),
                 ('ou:naics', {'subof': 'int', 'doc': 'North American Industry Classification System'}),
 
-                ('ou:suborg',
-                 {'subof': 'comp', 'fields': 'org,ou:org|sub,ou:org', 'doc': 'An org which owns a sub org'}),
-                ('ou:member', {'subof': 'comp', 'fields': 'org,ou:org|person,ps:person',
-                               'doc': 'A person who is (or was) a member of an organization.'}),
+                ('ou:suborg', {
+                    'subof': 'comp',
+                    'fields': 'org,ou:org|sub,ou:org',
+                    'doc': 'An org which owns a sub org'}),
```

Comment:
Suggest changing docstrings since suborg is broader than just "ownership". e.g., "Any "parent" / "child" relationship between two orgs. May represent ownership, organizational structure, a department within a larger entity, etc."